modules/ROOT/pages/dw-arrays-functions-takewhile.adoc

```diff
@@ -29,9 +29,9 @@ _Introduced in DataWeave 2.2.0. Supported by Mule 4.2 and later._
 === Example
 
 This example iterates over the elements in the array and selects only those
-where the index is `&lt;= 2`. It stops selecting elements when it reaches an 
-element that is greater than 2. Notice that it does not select 3 because 4 
-precedes 3 in the array. The function outputs the result into an array.
+where the index is `&lt;= 2`. Note that it stops selecting elements when it reaches an 
+element that is greater than 2. It does not select the second 1 because 4 
```

Comment:
Again with the "it." I'm stumped for a rewording...and maybe it's unnecessary. Up to you.